webapp/app/config.py

```diff
@@ -39,13 +39,18 @@ class BaseConfig(object):
 
     USE_LRU_CACHE = True
 
+    SESSION_DATA_REDIS_TTL_HOURS = 3
+
+    SESSION_DATA_STORAGE_URL = 'None'
```

Comment:
Can we delete this?

Reply:
I updated SESSION_DATA_STORAGE_URL so we access the env variable. A new variable was added: USE_REDIS_LITE with is default TRUE and only Stage/Prod have it to FALSE. 
Regarding SESSION_DATA_REDIS_TTL_HOURS it is a good location for that property, isn't it? Maybe we access that property somewhere else in the future